main.tf

```diff
@@ -128,7 +128,7 @@ resource "aws_cloudfront_distribution" "default" {
     acm_certificate_arn            = "${var.acm_certificate_arn}"
     ssl_support_method             = "sni-only"
     minimum_protocol_version       = "TLSv1"
-    cloudfront_default_certificate = true
+    cloudfront_default_certificate = "${var.acm_certificate_arn == "" ? 1 : 0}"
```

Comment:
nice catch @cliveza 

since `cloudfront_default_certificate` is boolean, should we change it to 
```hcl
cloudfront_default_certificate = "${var.acm_certificate_arn == "" ? true : false}
```

instead of `1 : 0` (although this would work too)